generators/client-2/templates/src/main/webapp/app/home/home.html

```diff
@@ -4,7 +4,7 @@
             <span class="hipster img-responsive img-rounded"></span>
         </div>
         <div class="col-md-8">
-            <h1 translate="home.title">Welcome, Java Hipster!</h1>
+            <h1 [jhi-translate]="'home.title' | translate">Welcome, Java Hipster!</h1>
```

Comment:
I dont like the markup its redundant. Let me see if there is a way to make it work kust with directive